app/components/sidebars/main/channels_list/list/list.js

```diff
@@ -144,8 +144,8 @@ export default class List extends PureComponent {
             return {
                 ...this.getSectionConfigByType(s.type),
                 data: s.items,
-                topSeparator: i !== 0 && orderedChannelIds[i - 1].items.length > 0,
-                bottomSeparator: true,
+                topSeparator: i !== 0,
+                bottomSeparator: s.items.length > 0,
```

Comment:
To confirm, isn't `s.items.length > 0` always true? (For the same reason we could simplify `topSeparator`.)